### DISCUSSION TITLE
[FTP] delete filed with wildcard *

### DISCUSSION THREAD

**Discussion by tome9111991** [curl/curl#7249] [Q&A] [answered]
Hello.
Ich possible to use curl to delete more files?
I want to dekete all files with a "."

-Q "DELE *.*" will not work.
Curl is not supporting MDELETE

Ist there a easy way to use wildcards?

**Comment by dfandrich** [comment, marked as answer]
-Q commands are sent to the server unprocessed. If the server supports
deleting wildcards, you can just use whatever syntax it uses. Likely, that be
an asterisk, meaning -Q "DELE .*" should work.

**Reply by tome9111991** [reply]
> -Q commands are sent to the server unprocessed. If the server supports deleting wildcards, you can just use whatever syntax it uses. Likely, that be an asterisk, meaning -Q "DELE .*" should work.

Is not work

550 .*: No such file or directory

**Comment by dfandrich** [comment]
You may actually want -Q "+DELE .*" so that curl sets the remote directory
before sending the command. If that doesn't do it, read the manual for your FTP
server to see if and how it supports deleting with wildcards. If it doesn't, 
you'd need to obtain a directory listing and delete the files individually.
